spotbugsTestCases/src/java/ghIssues/issue2370/Issue2370Doubles.java

```diff
@@ -0,0 +1,22 @@
+package ghIssues.issue2370;
+
+import java.util.Random;
+import java.util.stream.DoubleStream;
+
+public class Issue2370Doubles {
+  public DoubleStream shouldNotReport0() {
+    return new Random().doubles();
```

Comment:
Let me know if I should also add analogous tests for `SecureRandom` or if there is a preferred pattern for making these tests more DRY

Reply:
Should be OK as is.